src/ui/components/edit_user_pop_up/edit_user_pop_up.py

```diff
@@ -1,170 +1,110 @@
-            self._manager_icon = self.node.find_element(*MANAGER_ICON)
-        return self._manager_icon
-
-    @property
-    def user_type_button(self) -> WebElement:
-        return self.node.find_element(*USER_TYPE_BUTTON)
+        self.locators = {
+            **self.locators,
+            "edit_user_popup_title": ("xpath", ".//div[@class='edit-header']"),
+            "user_icon": ("xpath", "./descendant::div[@class='ellipse'][1]"),
+            "user_type_button": ("xpath", "//input[@value='ROLE_USER']/../.."),
+            "manager_icon": ("xpath", "./descendant::div[@class='ellipse'][2]"),
+            "manager_type_button": ("xpath", "//input[@value='ROLE_MANAGER']/../.."),
+            "last_name_input_element": ("xpath", "./descendant::div[contains(@class,'user-edit-input')][1]"),
+            "first_name_input_element": ("xpath", "./descendant::div[contains(@class,'user-edit-input')][2]"),
+            "phone_input_element": ("xpath", "./descendant::div[contains(@class,'user-edit-input')][3]"),
+            "phone_country_code": ("xpath", ".//span[@class='ant-input-group-addon']"),
+            "email_input_element": ("xpath", "./descendant::div[contains(@class,'user-edit-input')][4]"),
+            "upload_user_photo_element": ("xpath", ".//span[contains(@class, 'ant-upload-wrapper')]"),
+            "upload_user_photo_input": ("xpath", ".//input[@id='edit_urlLogo']"),
+            "photo_title": ("xpath", ".//label[@for='edit_urlLogo']"),
+            "question_circle_for_photo": ("xpath", ".//span[@aria-label='question-circle']"),
+            "photo_tooltip_form": ("xpath", "//div[@class='ant-tooltip-inner']"),
+            "checkbox_change_password_title": ("xpath", ".//div[contains(@class, 'align-checkbox')]/text"),
+            "checkbox_change_password_input": ("xpath", ".//input[@name='checkbox']"),
+            "current_password_input_element": ("xpath", "(//div[contains(@class,'item-control-input')]"
+                                                        "/span[contains(@class,'ant-input-password')"
+                                                        " and not (contains(@class,'login-box'))])[1]"),
+            "new_password_input_element": ("xpath", "(//div[contains(@class,'item-control-input')]"
+                                                    "/span[contains(@class,'ant-input-password')"
+                                                    " and not (contains(@class,'login-box'))])[2]"),
+            "confirm_password_input_element": ("xpath", "(//div[contains(@class,'item-control-input')]"
+                                                        "/span[contains(@class,'ant-input-password')"
+                                                        " and not (contains(@class,'login-box'))])[3]"),
+            "submit_button": ("xpath", ".//button[contains(@class, 'submit-button')]")
+        }
+        self._wait = WebDriverWait(self.driver, 30)
+        self._actions = ActionChains(self.driver)
```

Comment:
We already have methods get_wait() and get_actions() for this in the CustomPageFactory class

Reply:
Removed